projects/m5/001-budget-app/solutions/robertozecchini/budget.py

```diff
@@ -0,0 +1,87 @@
+class Category:
+    def __init__(self, name):
+        self.name = name
+        self.ledger = []
+    
+    def __str__(self):
+        s = self.name.center(30, '*') + "\n"
```

Comment:
So it looks that 's' is used presentational purposes.

2 Suggestions here:
- use a different naming convention: 's' is too generic and don't returns an immediate feedback of what is the purpose
- don't mix business logic and presentation: try to separate them by creating methods that just return values that could consumed by other presentational methods

Reply:
Ok, thanks. I will make another pull request trying to implement your suggestions.

Reply:
Ok, I have a technical issue: I have to make a new pull request or (once i have already loaded new commits to this branch) you can already see my new commits related to this pull request?